src/Utils.purs

```diff
@@ -36,21 +36,39 @@ inContainer :: ∀ item t f. (Unit -> t Unit) -> f -> Container.Query item t f
 inContainer = Container.ParentQuery <<< H.action
 
 getInputProps = augmentHTML
```

Comment:
@whoadave How would this work if we want to close the menu after a single selection? Right now, it keeps the menu open even if you click in on a child item. If the user wanted to instead have the menu close after any selection is made, what would that look like on their end (with only access to the render function)?